CONTRIBUTING.md

```diff
@@ -1,12 +1,27 @@
-It is recommended that you develop OmegaConf is a virtualenv or similar (pyenv, conda).
-To set up for development:
+### Setup development
+
+#### Essentials
+
+We reccomend using conda or pyenv to create a new environment. You can skip to install python 
+if you prefer. We'll use conda as an example:
+
+`conda create -n myenv python=3.6`
```

Comment:
1. no point in suggesting that they use system Python.
2. Please use 3.8 in the example. Even though OmegaConf supports Python 3.6 there is no point in using it unless there are issues.